spec/cjk/chinese_title_spec.rb

```diff
@@ -324,7 +324,7 @@
         let (:resp) { simp_resp }
       end
       it "ranks higher docs with one word in 245a and the other in 245b" do
-        #  9229845 - woman 245b, 246a; marriage 245a
+        #  9229845 - woman 245ab, 246a; marriage 245a
```

Comment:
so is woman now in 245b also?  If so, this may not be a good test object any more???  (bleah)